app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
+    @micropost.image.attach(params[:micropost][:image])
```

Comment:
You can use `params.dig()` to extract nested parameters. 
See https://apidock.com/rails/v6.0.0/ActionController/Parameters/dig